mlflow/store/model_registry/abstract_store.py

```diff
@@ -335,13 +334,17 @@ def copy_model_version(self, src_mv, dst_name):
         :return: Single :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
                  the cloned model version.
         """
-        pass
+        raise MlflowException(
+            "Method 'copy_model_version' has not yet been implemented for the current model "
+            "registry backend. To request support for implementing this method with this backend, "
+            "please submit an issue on GitHub."
+        )
```

Comment:
@harupy before I land this PR, does this error message look good? I discussed with @dbczumar to nudge users to file issues if they would like guidance or support implementing this method for their model registry backend.